.changeset/proud-phones-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-klaviyo": minor
+---
+
+Rewritten app to use Macaw UI / next
```

Comment:
I think we can have a more user-centric changeset here. I suggest including information about how rewriting the app with the new Macaw makes it align visually with the rest of the dashboard.

Reply:
Improved. But there are almost no changes so its hard describe anything interesting here